bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -188,6 +188,21 @@ spec:
           containerPort: {{.OVN_NB_PORT}}
         - name: nb-db-raft-port
           containerPort: {{.OVN_NB_RAFT_PORT}}
+        readinessProbe:
+          exec:
+            command:
+            - /bin/bash
+            - -c
+            - |
+              set -xe
+              OVN_NODES_ARRAY=({{.OVN_NODES}}) 
+              for node in "${OVN_NODES_ARRAY[@]}"; do
+                node_ip=$(getent ahostsv4 "${node}" | grep RAW | awk '{print $1}')
+                if ! ovs-appctl -t /var/run/openvswitch/ovnnb_db.ctl  cluster/status OVN_Northbound | grep -q $node_ip; then
+                  exit 1
+                fi
+              done
+              sleep 30
```

Comment:
I don't think you need this sleep. If you want to tweak the timing of readiness checks, there are a few specific knobs, see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#probe-v1-core

Reply:
This was only added because the US specified "Either way, a readiness probe that waits 30 seconds so rollouts are slow.". I can remove it though. I am not convinced this adds a lot of benefits. Why do we need to be slow when rolling out? And aren't there possible problems with ovnkube-node rolling out before ovnkube-master and having to wait for 30 seconds?